ambari-server/src/main/java/org/apache/ambari/server/controller/AmbariManagementControllerImpl.java

```diff
@@ -5145,35 +5146,69 @@ public synchronized LdapBatchDto synchronizeLdapUsersAndGroups(
     try {
 
       final LdapBatchDto batchInfo = new LdapBatchDto();
+      boolean postProcessExistingUsers = false;
+      boolean postProcessExistingUsersInGroups = false;
 
       if (userRequest != null) {
+        postProcessExistingUsers = userRequest.getPostProcessExistingUsers();
+
+        if(postProcessExistingUsers && !configs.isUserHookEnabled()) {
+          LOG.info("Post processing existing users is requested while processing users; however, the user post creation hook is turned off.");
```

Comment:
Minor issue:  

This looks like an error case, should the log level be ERROR?

Reply:
This is not really an error case, but I guess it depends on how to look at it.  I can turn this into a warning so it stands out.